src/pudl/settings.py

```diff
@@ -611,17 +611,20 @@ class EtlSettings(BaseSettings):
     pudl_in: str = pudl.workspace.setup.get_defaults()["pudl_in"]
     pudl_out: str = pudl.workspace.setup.get_defaults()["pudl_out"]
 
+    # This is list of fsspec compatible paths to publish the output datasets to.
+    publish_destinations: list[str] = []
```

Comment:
Do we need set some destinations in the settings files so that the nightly build script still publishes?

Reply:
No, the current functionality (publishing via wrapper shell script) should be unaffected. Eventually we'll want to switch to the "new" mode and drop the shell script stuff, but for now, we keep relying on the old method.